src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const allLi = document.querySelectorAll('li');
+const tree = document.querySelector('.tree');
+
+for (const li of allLi) {
+  if (li.children.length !== 0) {
+    const span = document.createElement('span');
+
+    li.prepend(span);
+    span.append(span.nextSibling);
+  }
```

Comment:
This block of code is responsible for wrapping the text of each `li` element with a `span`. However, it only does so if the `li` element has children. This means that `li` elements without children will not have their text wrapped in a `span`, and therefore clicking on their text will not trigger the click event listener on the tree. To ensure that the click event listener works for all `li` elements, consider removing the condition `if (li.children.length !== 0)` and always creating and appending the `span`.